milestone-5/backend/server.ts

```diff
@@ -0,0 +1,23 @@
+import { Express } from "express";
+import mongoose from "mongoose";
+import recipeEndPoints from "./routes/recipe";
+
+const express = require("express"); // 1. includes Express
+const app: Express = express(); // 2. initializes Express
+const cors = require("cors");
+app.use(express.json()); // Enables JSON parsing
+app.use(cors());
+
+console.log("Backend running");
+
+const url =
+  "mongodb+srv://admin:7cKIwO4vYk2zZeSY@cluster0.0b736wr.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
Hey Sam, It appears you forgot to add [open network access ](https://www.notion.so/h4i/Getting-started-with-MongoDB-Atlas-a385adf1b2b44a5bb473553700cad088#ec0f63d8e9234cd8b2b47d9ac9be80fd)to your mongoDB cluster. It won't let me run your backend on my ip address so I can't test your code. Could you add this and resubmit so I can get you checked off for all the milestones.

Reply:
Hi Sam,
Sorry I missed that part! Should be added now. Thanks!